grails-app/assets/javascripts/permissionTable_original.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Render project members and their roles, support pagination.
```

Comment:
Is this a backup that was accidentally committed?

Reply:
This is the original implementation I used last year, I will remove it from the repo.